### PR TITLE
Make all static pages mobile-friendly

### PR DESCRIPTION
**Pull request by stvstnfrd**

### CODE REVIEW

File: lagunita/lms/static/sass/partials/base/custom_overrides/_header.scss [hunk 1]
```diff
@@ -51,3 +51,44 @@
         }
     }
 }
+section.container {
```
Comment (stvstnfrd) [inline lagunita/lms/static/sass/partials/base/custom_overrides/_header.scss]: Oh, I need to move this somewhere better.
I just stuck it here to get started.

Comment (caesar2164) [reply lagunita/lms/static/sass/partials/base/custom_overrides/_header.scss]: maybe add a new `_mobile.scss`?

File: lagunita/lms/templates/static_templates/contact.html [hunk 2]
```diff
@@ -13,13 +13,9 @@
 <%block name="pagetitle">${_('Contact')}</%block>
 
 <main id="main" aria-label="Content" tabindex="-1">
-    <section class="container about contact">
+    <section class="container contact">
         <section class="vision">
             <section class="message left" id="contact">
-                <div class="photo">
```
Comment (caesar2164) [inline lagunita/lms/templates/static_templates/contact.html:19]: why are removing the dish photo?

Comment (stvstnfrd) [reply lagunita/lms/templates/static_templates/contact.html:19]: Eh, why not?
The about page looks nice spruced up w/ an image, but this is just contact info, so one less distraction?
I mean, we could include it inline, but doesn't feel like it adds much to the page. It's cute and all, the dish for contact ;)

File: lagunita/lms/static/sass/partials/base/custom_overrides/_footer.scss [hunk 3]
```diff
@@ -5,7 +5,7 @@
         .wrapper-logo {
             width: $footer-logo-width;
             float: left;
-            margin: 0 25px 0 0;
+            margin: 0 15px 15px 0;
```
Comment (caesar2164) [inline lagunita/lms/static/sass/partials/base/custom_overrides/_footer.scss:8]: I agree with this change :)

File: lagunita/lms/static/sass/partials/base/_overrides.scss [hunk 4]
```diff
@@ -22,3 +22,5 @@ $footer-logo-width: 100px;
 .xblock .mentoring .copyright {
     display: none;
 }
+
+@import 'custom_overrides/_mobile.scss';
```
Comment (caesar2164) [inline lagunita/lms/static/sass/partials/base/_overrides.scss:26]: Isn't it nice getting to keep this file so nice and clean?

**Review by caesar2164 — APPROVED** [review]

**Review by caesar2164 — APPROVED** [review]
👍

**Pull request merged**